apps/web/src/views/Home/components/Banners/GalxePedictBanner.tsx

```diff
@@ -0,0 +1,168 @@
+    margin-bottom: 10px;
+    &::after {
+      padding-right: 0px;
+    }
+  }
+`
+
+const GalxePredictBanner = () => {
+  const { t } = useTranslation()
+  const { isMobile } = useMatchBreakpoints()
+
+  return (
+    <S.Wrapper
+      style={{
+        background: 'radial-gradient(circle, rgba(66,159,169,1) 0%, rgba(89,58,151,1) 100%)',
+      }}
+    >
+      <S.Inner>
+        <S.LeftWrapper position="relative" style={{ zIndex: 2, justifyContent: isMobile ? 'flex-start' : 'center' }}>
+          <Flex alignItems="center" style={{ gap: isMobile ? 4 : 12 }} mb="8px">
+            <Image
+              src={pancakeSwapLogo}
+              alt="pancakeSwapLogo"
+              width={isMobile ? 90 : 132}
+              height={isMobile ? 13 : 20}
+              unoptimized
+            />
+          </Flex>
+          <Box maxWidth="780px" marginTop="12px">
+            <StyledSubheading
+              data-text={isMobile ? t('Jupiter Predictors:') : t('Predict, win and share $10,000 rewards')}
+            >
+              {isMobile ? t('Jupiter Predictors:') : t('Predict, win and share $10,000 rewards')}
+            </StyledSubheading>
+          </Box>
+
+          <Flex marginTop={isMobile ? '24px' : '12px'}>
+            <Link style={{ textDecoration: 'none' }} external href="https://pancakeswap.finance/prediction">
+              <Text
+                textTransform={isMobile ? 'uppercase' : 'capitalize'}
+                bold
+                fontSize={isMobile ? '12px' : '16px'}
+                mr="4px"
+                color="white"
+              >
+                {t('Predict now')}
+              </Text>
+            </Link>
+            <Text
+              textTransform={isMobile ? 'uppercase' : 'capitalize'}
+              bold
+              fontSize={isMobile ? '12px' : '16px'}
+              mr="4px"
+              color="white"
+              px="8px"
+            >
+              |
+            </Text>
+            <Link
+              style={{ textDecoration: 'none' }}
+              external
+              href="https://blog.pancakeswap.finance/articles/pancake-swap-s-de-fi-galaxy-tour-planet-5-jupiter-predictors-predicting-crypto-movements"
+            >
+              <Text
+                textTransform={isMobile ? 'uppercase' : 'capitalize'}
+                bold
+                fontSize={isMobile ? '12px' : '16px'}
+                mr="4px"
+                color="white"
+              >
+                {t('Learn more')}
+              </Text>
+              <OpenNewIcon color="white" />
+            </Link>
+          </Flex>
+        </S.LeftWrapper>
+        <RightWrapper>
+          <BgWrapper>
+            {!isMobile ? (
+              <Image src={galxePredictorsBg} height={192} alt="liquidStakingBunny" placeholder="blur" />
+            ) : (
+              <Image src={galxeBirthdayCampaignPedictMobile} height={192} alt="liquidStakingBunny" placeholder="blur" />
+            )}
+          </BgWrapper>
```

Comment:
Size weird. add width.
And use x2 image

Reply:
width added